CMakeLists.txt

```diff
@@ -216,8 +216,12 @@ set(MD_VERSION ${MD_VERSION})
 configure_file(libmd.pc.in ${CMAKE_BINARY_DIR}/libmd.pc @ONLY)
 install (FILES ${CMAKE_BINARY_DIR}/libmd.pc DESTINATION lib/pkgconfig)
 
-install (TARGETS ${PROJECT_NAME} DESTINATION lib)
-install (FILES "${BS_DIR_INC}/common/instrumentation/api/metrics_discovery_api.h" DESTINATION include)
+# Library/header
+if (${LIB_INSTALL_DIR} STREQUAL "")
+     set(LIB_INSTALL_DIR lib${LIB_SUFFIX})
```

Comment:
I see one problem with using LIB_SUFFIX like this. After providing LIB_SUFFIX to cmake, installed pkg-config file (libmd.pc) will have wrong libdir specified.

Reply:
@litcoder  libmd.pc.in has ${prefix}/**lib** hardcoded